model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -1362,4 +1347,27 @@ message ExecutableStagePayload {
     // (Required) The local name of this timer for the PTransform that references it.
     string local_name = 2;
   }
+
+  // Settings that decide the coder type of wire coder.
+  message WireCoderSetting {
+    // (Required) The URN of the wire coder.
+    // Note that only windowed value coder or parameterized windowed value coder are supported.
+    string urn = 1;
+
+    // (Optional) The data specifying any parameters to the URN. If
+    // the URN is beam:coder:windowed_value:v1, this may be omitted. If the URN is
+    // beam:coder:param_windowed_value:v1, the payload is an encoded windowed
+    // value using the beam:coder:windowed_value:v1 coder parameterized by
+    // a beam:coder:bytes:v1 element coder and the window coder that this
+    // param_windowed_value coder uses.
+    bytes payload = 2;
+
+    // (Required) The target(PCollection or Timer) this setting applies to.
+    oneof target {
```

Comment:
![image](https://user-images.githubusercontent.com/22488084/71609233-e7735800-2bc1-11ea-88cd-bc7cf547b964.png)

Reply:
Hi @lukecwik. Do you think we should add the WireCoderSetting property for each input and output edge explicitly in the proto? 
I was thinking the setting is not a necessary properties for each edge so added the wire_coder_settings as an ExecutableStage level property. The wire_coder_settings can be an empty list. 
What do you think?